src/blueprint/into_schema.rs

```diff
@@ -68,8 +69,10 @@ fn to_type(def: &Definition) -> dynamic::Type {
                                         let ctx: ResolverContext = ctx.into();
                                         let ctx = EvaluationContext::new(req_ctx, &ctx);
 
-                                        let const_value =
-                                            expr.eval(ctx, &Concurrent::Sequential).await?;
+                                        let const_value = expr
+                                            .eval(ctx, &Concurrent::Sequential)
+                                            .await
+                                            .map_err(|err| err.extend())?;
```

Comment:
Just a thought: We could use `into` here. Using `extend` doesn't seem very intuitive.

Reply:
no, `into` will use implicit implementation for Error https://docs.rs/async-graphql/latest/src/async_graphql/error.rs.html#322-330 that loses additional information